README.md

```diff
@@ -9,32 +9,53 @@
 Please make sure you have [**Pieces OS**](https://docs.pieces.app/installation-getting-started/what-am-i-installing) installed to run the Package.
 
 
-## Getting Started
```

Comment:
The Pieces Sublime Plugin logo should have the image similar to other plugins: https://github.com/pieces-app/obsidian-pieces/raw/main/assets/readme/pfd-obisidan-plugin-hero.png

Reply:
@laurin-at-pieces can provide the image for Sublime plugin